### ISSUE TITLE
crashing on 1.16.2

### ISSUE THREAD

**Issue by Guil-W** [hrydgard/ppsspp#18177]
### Game or games this happens in

ULUS-10512 - Shin Megami Tensei: Persona 3 Portable

### What area of the game / PPSSPP

at startup

### What should happen

it shouldn't crash on startup, the previous version didn't do that! 

### Platform

Android

### Mobile phone model or graphics card

LG K22+

### PPSSPP version affected

1.16.2

### Last working version

last time i used PPSSPP, the version was 1.13.2, so i don't know the exact version.

### Graphics backend (3D API)

OpenGL / GLES

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
https://www.gsmarena.com/lg_k22-10597.php

Those are some bad specs, but I haven't gotten any other reports of this game crashing recently.

Can you try 1.14.4 and 1.15.4, to start with, to narrow down the search?

**Comment by Guil-W** [comment]
alright alright

**Comment by Guil-W** [comment]
> https://www.gsmarena.com/lg_k22-10597.php
> 
> Those are some bad specs, but I haven't gotten any other reports of this game crashing recently.
> 
> Can you try 1.14.4 and 1.15.4, to start with, to narrow down the search for what broke it on your phone?

quick update, it works when i try the software renderer, but it doesnt run textures + it's too slow, although Final Fantasy 1 20th Anniversary runs silky smooth

**Comment by Guil-W** [comment]
with the hardware renderer i mean

**Comment by ANR2ME** [comment]
Btw are you using texture replacement? 
If you are, does it crashes too without texture replacement? (may be you're running out of vram)

**Comment by Guil-W** [comment]
yeah, removed textures, still freeze frames, and boots back to app select menu

**Comment by Guil-W** [comment]
there must be a hidden setting under my nose, otherwise idk

**Comment by Guil-W** [comment]
> https://www.gsmarena.com/lg_k22-10597.php
> 
> Those are some bad specs, but I haven't gotten any other reports of this game crashing recently.
> 
> Can you try 1.14.4 and 1.15.4, to start with, to narrow down the search for what broke it on your phone?

i also tried 1.14.4, no dice...

**Comment by unknownbrackets** [comment]
Hm, sounds to me like #16015, maybe it's back to crashing with some change?

-[Unknown]
